crates/containerd-shim-wasm/Cargo.toml

```diff
@@ -36,7 +36,10 @@ wasmparser = "0.207.0"
 tokio-stream = { version = "0.1" }
 prost-types = "0.12" # should match version in containerd-shim
 sha256 = { workspace = true }
-tracing = { workspace = true }
+
+# tracing
+shim-instrument = { git = "https://github.com/Mossaka/shim-instrument", branch = "main" }
```

Comment:
I am upstreaming this implementation in rust-extension (see https://github.com/containerd/rust-extensions/pull/269), and as soon as that PR gets approved and merged, I will change the git link here and make this PR ready for review.